pkg/controller/cabundleinjector/mutatingwebhook.go

```diff
@@ -57,7 +57,7 @@ func (bi *mutatingWebhookCABundleInjector) Sync(ctx context.Context, syncCtx fac
 
 	// make a copy to avoid mutating cache state
 	webhookConfigCopy := webhookConfig.DeepCopy()
-	for i := range webhooksNeedingUpdate {
```

Comment:
Thanks for the fix! Please add a unit test.

Reply:
Hi @stlaz 

I added a unit test on my fork here, but don't have permissions to the fork in this PR. https://github.com/sam6258/service-ca-operator/commit/bce2b38b10a91af70c5df4641a9c1401b37e7b57

Unfortunately @dunnevan has left our company, but I would like to continue to drive this fix. I can open a new PR if needed or if possible, you could redirect this one to my fork.

Thanks!

Reply:
Hey Scott, I pulled in your commit to this branch.  I also invited you to collaborate on my fork if you need to push anything else.